### ISSUE TITLE
Feature request: allow to choose a ship for voyage when multiple have the same AM

### ISSUE THREAD

**Issue by johanw666** [IAmPicard/StarTrekTimelinesSpreadsheet#157]
**Is your feature request related to a problem? Please describe.**
Whan I send a voyage and I have multiple ships with the same amount of AM for this voyage, always the first one is choosen.

**Describe the solution you'd like**
A method for me to choose any of the ships with the same amount of AM.

**Describe alternatives you've considered**
None come to mind.

**Comment by lobalope** [comment]
Yes! The ability to choose from the Best Ship(s) listed would be awesome!
